src/components/room/index.tsx

```diff
@@ -123,7 +123,13 @@ function Room({id: roomId, peer, username, initiator}: IRoom) {
     };
 
     const onHangUp = () => {
+        const videoTracks = state.localStream.getVideoTracks() || [];
         peer.disconnect();
+        videoTracks.forEach((track: MediaStreamTrack) => {
```

Comment:
We should stop the tracks before closing all connections so it doesn't appear like the video has frozen for the other peers.

```suggestion
        tracks.forEach((track: MediaStreamTrack) => {
```